packages/components/src/mobile/global-styles-context/utils.native.js

```diff
@@ -231,11 +246,11 @@ export function parseStylesVariables( styles, mappedValues, customValues ) {
 						customValuesData
 					)
 				) {
-					return get( customValuesData, path );
+					return getValueFromObjectPath( customValuesData, path );
 				}
 
 				// Check for camelcase properties.
-				return get( customValuesData, [
+				return getValueFromObjectPath( customValuesData, [
```

Comment:
This is as safe as `get()` was, because `getValueFromObjectPath()` will handle nullish values.